src/post/components/PostsList/PostsLists.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from "@testing-library/react";
+import { Post } from "../../types";
+import PostsList from "./PostsList";
+
+describe("Given the PostsList component", () => {
+  describe("When it receives two posts with the title 'hola buenas' and 'hasta luego cocodrilo'", () => {
+    test("Then it should show 'hola buenas' and 'hasta luego cocodrilo' inside headings", () => {
+      const posts: Post[] = [
+        {
+          id: "",
+          title: "hola buenas",
+          author: "",
+          date: new Date(),
+          content: "",
+        },
+        {
+          id: "",
+          title: "hasta luego cocodrilo",
+          author: "",
+          date: new Date(),
+          content: "",
+        },
+      ];
+      render(<PostsList posts={posts} />);
```

Comment:
**UN**
**IN**
**TRO**

Reply:
Mario, mira el último commit, justo lo he corregido en la PR que me lo ha revisado @lexisgasa